tests/scripts/start-storybook.sh

```diff
@@ -1,21 +1,28 @@
 #!/bin/bash
 
-npm run storybook -- --quiet --ci &
-JOB_ID=$(echo $!) #get the background job ID
+#check for existing process on port:6006
+process=`lsof -i :6006`
```

Comment:
wait actually sorry one last thing, can we rename this variable? process is a little vague

Reply:
maybe preexistingPortProcess? I don't love the name I came up with so open to suggestions!

Reply:
sounds good to me!